crates/sum_tree/src/sum_tree.rs

```diff
@@ -113,6 +151,10 @@ impl Bias {
     }
 }
 
+/// A binary tree where each leaf node contains an [`Item`] of type `T`,
```

Comment:
This is incorrect, a SumTree is a B-Tree, not a binary tree

Reply:
Good catch, thank you!